include/rcpputils/filesystem_helper.hpp

```diff
@@ -43,6 +43,9 @@
 #ifndef RCPPUTILS__FILESYSTEM_HELPER_HPP_
 #define RCPPUTILS__FILESYSTEM_HELPER_HPP_
 
+#include <sys/stat.h>
+#include <sys/types.h>
```

Comment:
Are these POSIX specific? Maybe they belong below in the conditional block for includes.

https://stackoverflow.com/questions/25280440/why-include-direct-h-or-sys-stat-h-conditionally-based-on-win32-or-linux

Reply:
I fixed it, although, not sure `sys/types.h` is needed. Let me know if it looks ok :)